pom.xml

```diff
@@ -84,7 +84,7 @@
 		<dependency>
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
-			<version>2.4</version>
+			<version>2.1</version>
```

Comment:
It's a quick fix to make the code compile, since org.irods.jargon depends on commons-io 2.1. However, we can remove irods related code now.